resources/views/member/dashboard.blade.php

```diff
@@ -2,26 +2,365 @@
 
 @section('content')
     <!-- page content -->
+    <div class="clearfix"></div>
 
-    <div class="row">
-        <div class="col-md-12 col-sm-12 col-xs-12">
-            <div id="log_activity" class="dashboard_graph">
+    <h1 class="username"><span data-username></span> - <span data-company></span></h1>
 
-                <div class="row x_title">
-                    <div class="col-md-6">
-                        <h3>{{ __('views.member.dashboard.sub_title_0') }}</h3>
+    <div class="container">
+        <div class="row">
+            <div class="col-xs-12 col-sm-6 col-lg-4">
+                <div class="x_panel">
+                    <div class="x_title clearfix">
+                        <h2>Revenue</h2>
```

Comment:
Please use embedded translation feature `{{__('views.member.revenue')}}`. This should be added to en.json